File: src/main/java/ua/softgroup/matrix/server/persistent/repository/TrackingDataRepository.java
package ua.softgroup.matrix.server.persistent.repository;

import org.springframework.data.repository.CrudRepository;
import ua.softgroup.matrix.server.persistent.entity.TrackingData;
import ua.softgroup.matrix.server.persistent.entity.WorkDay;

public interface TrackingDataRepository extends CrudRepository<TrackingData, Long> {

    TrackingData findByWorkTimePeriodWorkDay(WorkDay workDay);

}

File: src/main/scala/ua/softgroup/matrix/server/persistent/repository/TrackingDataRepository.scala
package ua.softgroup.matrix.server.persistent.repository

import org.springframework.data.repository.CrudRepository
import ua.softgroup.matrix.server.persistent.entity.{TrackingData, WorkDay}

/**
  * @author Oleksandr Tyshkovets <sg.olexander@gmail.com> 
  */
trait TrackingDataRepository extends CrudRepository[TrackingData, java.lang.Long] {

  def findByWorkTimePeriodWorkDay(workDay: WorkDay): TrackingData

}
